### PR TITLE
Adding guides for bug reports and feature requests

### PR DESCRIPTION
**Pull request by CatGarab**

### CODE REVIEW

**Comment by CatGarab** [comment]
[Docs on reporting bugs/doing feature requests](https://trello.com/c/D6d1Bnex/691-docs-on-reporting-bugs-doing-feature-requests)

File: docs/contributing/feature-request.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+If the feature request is not accepted, the reason for the rejection will be recorded and the issue
+will be closed.
+A request could be declined because a duplicate request already exists, or because the Kubos team
+feels that it does not match the product's goals.
+
+If the feature request is accepted, there will likely be some back and forth between you and/or
+Kubos team members in order to determine the specifics of how the feature request should be
+implemented and the low-level desired behavior.
+
+The team (or potentially a community member!) will then implement the changes needed to fufill the
```
Comment (cantino) [inline docs/contributing/feature-request.rst]: *fulfill*

Comment (cantino) [reply docs/contributing/feature-request.rst]: *(or potentially you or another community member!)*

File: docs/contributing/reporting-bugs.rst [hunk 2]
```diff
@@ -1,6 +1,71 @@
 Reporting Bugs
 ==============
 
-Pffffft, our code is perfect. No bugs here.
+While all devs like to think that they write perfect code, we know that reality doesn't quite agree.
 
-TODO. It's actually its own card in the backlog.
\ No newline at end of file
+If you encounter a bug while using KubOS or while working with our source code, please open
```
[the remark this answers is not in the document]
Comment (CatGarab) [reply docs/contributing/reporting-bugs.rst]: :+1: I added a note in the bug report doc

**Pull request merged**